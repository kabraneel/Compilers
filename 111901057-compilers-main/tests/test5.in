x := 3;
y := 4;
for i:= 2 to 70 
do
	print x+y;
done;
