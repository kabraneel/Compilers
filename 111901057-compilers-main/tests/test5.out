.globl main
main:
li $t1, 3
li $t2, 4
li $t3, 2
li $t4, 70
label1:
beq $t3, $t4, label2
move $t6, $t1
move $t7, $t2
add $t5, $t6, $t7
move $a0, $t5
li $v0, 1
syscall
addi $t3, $t3, 1
j label1
label2:
li $v0, 10
syscall
